src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
+
 public class Duke {
+
+    public static void printMessage(String text) {
+        System.out.println("\n____________________________________________________________");
+        System.out.println(text);
+        System.out.println("____________________________________________________________\n");
```

Comment:
Maybe you can combine the three print functions into one
ie:
System.out.println("\n____________________________________________________________\n"
        + text + "\n____________________________________________________________\n");